### DISCUSSION TITLE
[npm ERR!] can't run typescript projects

### DISCUSSION THREAD

**Discussion by kyzens** [community/community#75915] [Programming Help] [answered]
### Body

`npm start `
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path D:\My Project\Latihan WEB\basic-typescripts/package.json
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open 'D:\My Project\Latihan WEB\basic-typescripts\package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\coccl\AppData\Local\npm-cache\_logs\2023-11-18T04_36_21_688Z-debug-0.log

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by wayosu** [comment, marked as answer]
add it to package.json in the  scripts section:
`"start": "tsc && node bundle/index.js"`

then try running `npm run start`

**Reply by kyzens** [reply]
woah thanks man. the project can already run 👍

**Reply by wayosu** [reply]
glad I could help bro 👌
